
public class ExceptionExample1 {

	public static void main(String[] args) {
		
		try
		{
			return;
		}
		finally
		{
			System.out.println("fanally block");//out put is finally block
		}

	}

}
